lib/plug/csrf_protection.ex

```diff
@@ -173,6 +173,7 @@ defmodule Plug.CSRFProtection do
   ## Plug
 
   @behaviour Plug
+  # "HS256" (HMAC using SHA-256 hash algorithm) encoded in Base64.
```

Comment:
I am very ignorant on what's happening here, but why:

```
iex(1)> Base.encode64("HS256")
"SFMyNTY="
```

? Why is the padding `=` in our implementation but `.` in `@digest` here?

Reply:
From [here](https://github.com/fertapric/plug/blob/b01c3bd55f6b21d58080791fef054e581c23fb97/lib/plug/crypto/message_verifier.ex#L125)?

Reply:
Yeah, definitely from there:

```
iex(1)> "HS256" |> Base.url_encode64(padding: false) |> Kernel.<>(".")
"SFMyNTY."
```

Reply:
Ah, nice, thanks. Maybe it would be good to encode this value at compile time instead of using the result directly? I'm talking about having:

```elixir
@digest Base.url_encode64("HS256", padding: false) <> "."
```

Thoughts?

Reply:
It sounds good to me

Reply:
Can you send a PR @fertapric? :)